### ISSUE TITLE
Crashing page

### ISSUE THREAD

**Issue by Snazzie** [benlhachemi/reactemojis#2]
Hello! im trying to use your package but as soon as its included in the page, it crashes the entire page.

`<ReactEmojis emoji="🚀" />`

![msedge_YxxupGBIOZ](https://github.com/user-attachments/assets/fbb10548-cfb3-4d8e-8495-333df44ce8d6)

Im using docusaurus v3

```
Uncaught Error: Element type is invalid. Received a promise that resolves to: [object Object]. Lazy element type must resolve to a class or function.
    at mountLazyComponent (react-dom.development.js:20042:1)
    at beginWork (react-dom.development.js:21632:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:4164:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4213:1)
    at invokeGuardedCallback (react-dom.development.js:4277:1)
    at beginWork$1 (react-dom.development.js:27490:1)
    at performUnitOfWork (react-dom.development.js:26599:1)
    at workLoopConcurrent (react-dom.development.js:26582:1)
    at renderRootConcurrent (react-dom.development.js:26544:1)
    at performConcurrentWorkOnRoot
```
    
```
   The above error occurred in one of your React components:

    at Suspense
    at Emoji (webpack-internal:///./node_modules/.pnpm/@souhaildev+reactemojis@1.0.5_react-dom@18.3.1_react@18.3.1_/node_modules/@souhaildev/reactemojis/dist/cool-GVRHLWML.mjs:25076:16)
    at Suspense
    at ReactEmojis (webpack-internal:///./node_modules/.pnpm/@souhaildev+reactemojis@1.0.5_react-dom@18.3.1_react@18.3.1_/node_modules/@souhaildev/reactemojis/dist/index.mjs:10:22)
    at section
    at FeaturesSection
    at main
    at ErrorBoundary (webpack-internal:///./node_modules/.pnpm/@docusaurus+core@3.5.2_@docusaurus+types@3.5.2_react-dom@18.3.1_react@18.3.1__react@18.3.1__@_xnphdcqkygbput5jugphfsqp74/node_modules/@docusaurus/core/lib/client/exports/ErrorBoundary.js:16:269)
    at div
    at NavbarSecondaryMenuDisplayProvider (webpack-internal:///./node_modules/.pnpm/@docusaurus+theme-common@3.5.2_@docusaurus+plugin-content-docs@3.5.2_@mdx-js+react@3.0.1_@typ_3gz7xfkyinmkyoc46joza5nmvu/node_modules/@docusaurus/theme-common/lib/contexts/navbarSecondaryMenu/display.js:21:318)
    at NavbarMobileSidebarProvider (webpack-internal:///./node_modules/.pnpm/@docusaurus+theme-common@3.5.2_@docusaurus+plugin-content-docs@3.5.2_@mdx-js+react@3.0.1_@typ_3gz7xfkyinmkyoc46joza5nmvu/node_modules/@docusaurus/theme-common/lib/contexts/navbarMobileSidebar.js:23:414)
```

```
    Element type is invalid. Received a promise that resolves to: [object Object]. Lazy element type must resolve to a class or function.
    at mountLazyComponent (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:20037:9)
    at beginWork (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:21627:16)
    at HTMLUnknownElement.callCallback (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:4164:14)
    at Object.invokeGuardedCallbackDev (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:4213:16)
    at invokeGuardedCallback (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:4277:31)
    at beginWork$1 (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:27485:7)
    at performUnitOfWork (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:26594:12)
    at workLoopConcurrent (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:26577:5)
    at renderRootConcurrent (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:26539:7)
    at performConcurrentWorkOnRoot (webpack-internal:///./node_modules/.pnpm/react-dom@18.3.1_react@18.3.1/node_modules/react-dom/cjs/react-dom.development.js:25772:3
```

**Comment by benlhachemi** [comment]
Hello,
make sure to use the 'use client' directive,
also most of the times the page crashes if you import all the animations at once (which what happens when you import ReactEmojis)

If you encounter performance issues with the package, and you notice it's running slowly, you can enhance it by selectively importing only the emojis you need. Follow these steps:

Instead of the generic import:

```
import ReactEmojis from "@/souhaildev/reactemojis";
<ReactEmoji emoji="🔥" />
```


Opt for a specific emoji import:

```
import FireEmoji from "@/souhaildev/reactemojis/components/fire";
<FireEmoji />
```

To recap, import only the required emoji by following these steps:

1 - Identify the name of the desired emoji from the [Emoji List](https://docs.reactemojis.com/docs/list-of-emojis).
2 - Import the emoji component using the format:

`import Emoji from "@/souhaildev/reactemojis/components/name-of-emoji-here";`


This approach allows you to optimize the package's performance by loading only the necessary components.

Docs: [https://docs.reactemojis.com/docs/tutorials/how-to-optimize](https://docs.reactemojis.com/docs/tutorials/how-to-optimize)
